### PR TITLE
CC-266: use default values from Schema in DDL

### PR DESCRIPTION
**Pull request by shikhar**

### CODE REVIEW

**Comment by shikhar** [comment]
@confluentinc/connect ptal

File: src/test/java/io/confluent/connect/jdbc/sink/dialect/DbDialectTest.java [hunk 1]
```diff
@@ -16,9 +16,13 @@
 
 package io.confluent.connect.jdbc.sink.dialect;
 
+import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.errors.ConnectException;
 import org.junit.Test;
 
+import java.nio.charset.Charset;
```
Comment (ewencp) [inline src/test/java/io/confluent/connect/jdbc/sink/dialect/DbDialectTest.java]: unused imports

File: src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java [hunk 2]
```diff
@@ -107,13 +110,50 @@ protected void writeColumnSpec(StringBuilder builder, SinkRecordField f) {
...
+      formatColumnValue(builder, f.type, f.defaultValue);
+    } else if (f.isOptional) {
       builder.append(" NULL");
     } else {
       builder.append(" NOT NULL");
     }
   }
 
+  static void formatColumnValue(StringBuilder builder, Schema.Type type, Object value) {
+    switch (type) {
```
Comment (ewencp) [inline src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java]: Should we be checking for any of the logical types?

**Comment by ewencp** [comment]
@shikhar One nit, one question about supporting logical types. Otherwise LGTM

File: src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java (hunk 2 above)
Comment (shikhar) [inline src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java]: Good point. We don't support logical types elsewhere yet (in DDL `DbDialect.getSqlType()`, for insertion `PreparedStatementBinder.bindField`), so not worth starting here.

We should, though! Not sure if it's important for the initial release. I'll ticket it.

Comment (shikhar) [inline src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java]: https://confluentinc.atlassian.net/browse/CC-278

**Pull request merged**